src/components/common/NameRequestInfo.vue

```diff
@@ -74,31 +74,6 @@
       </div>
     </template>
 
-    <!-- display changed name info -->
-    <template v-else-if="getNameRequestApprovedName">
```

Comment:
Some amalgamation options used this code instead of their specific code block, so I simply moved this below the amalgamation options.